src/Logging/Logging.Abstractions/src/NullLogger.cs

```diff
@@ -10,6 +10,9 @@ namespace Microsoft.Extensions.Logging.Abstractions
     /// </summary>
     public class NullLogger : ILogger
     {
+        /// <summary>
+        /// Returns an instance of <see cref="NullLogger"/>.
```

Comment:
Not just `an` instance. It's `the` instance. Maybe, `Returns the shared instance of BLAH`?